frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1954,6 +1961,7 @@ class Worksheet extends React.Component {
                     toggleCmdDialog={this.toggleCmdDialog}
                     toggleInformationModal={this.toggleInformationModal}
                     toggleCmdDialogNoEvent={this.toggleCmdDialogNoEvent}
+                    openErrorMessageDialog={this.openErrorMessageDialog}
```

Comment:
I think it may be clearer to rename this function to onError.

Reply:
updated